src/BitOasis/Coin/Address/LitecoinAddress.php

```diff
@@ -121,7 +121,7 @@ public function toLegacyAddressFormat() {
 			return new static(Base58Check::encodeAddress($decodedAddress), $this->currency);
 		}
 		
-		return $this;
+		return null;
```

Comment:
You should not change current API of library. Or have you checked all projects if it won't do any bc break?